### PR TITLE
slof_greater_lun_id: Add slof_greater_lun_id case for testing SLOF

### PR DESCRIPTION
**Pull request by YongxueHong**
slof_greater_lun_id case is that verify SLOF info
when LUN ID greater than 255.

ID: 1645020

Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
Depends on https://github.com/autotest/tp-qemu/pull/1531

**Comment by wangxianxian** [comment]
python3 ConfigTest.py --driveformat=virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --guestname=RHEL.x.x --platform=ppc64le --mem=4096 --vcpu=4 --testcase=slof_greater_lun_id --clone=no
qemu.slof_greater_lun_id: PASS

**Comment by wangxianxian** [comment]
Acked by xianwang@redhat.com

**Comment by YongxueHong** [comment]
@lolyu  @hereischen  help review, thanks.

File: qemu/tests/slof_greater_lun_id.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    params['start_vm'] = 'yes'
+    start_pos = 0
+    lun1 = params['boot_lun_id']
+    lun2 = params['reboot_lun_id']
+    for params['drive_lun_image1'] in (lun1, lun2):
```
Comment (hereischen) [inline qemu/tests/slof_greater_lun_id.py]:
```python
    luns = (params['boot_lun_id'], params['reboot_lun_id'] )
    for params['drive_lun_image1'] in luns:
```

Comment (YongxueHong) [reply qemu/tests/slof_greater_lun_id.py]: thanks, have updated.

File: qemu/tests/cfg/slof_greater_lun_id.cfg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    start_vm = no
+    not_preprocess = yes
+    only ppc64le ppc64
+    only Linux
+    only virtio_scsi
+    # scsi-hd attached spapr_vscsi bus support the maximum LUN ID is 7.
+    no spapr_vscsi
+    drive_format_image1 = scsi-hd
+    boot_lun_id = 0
+    reboot_lun_id = 300
```
Comment (lolyu) [inline qemu/tests/cfg/slof_greater_lun_id.cfg]: suggest using `boot_lun_ids = 0,300`

File: qemu/tests/slof_greater_lun_id.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+        error_context.context("Try to log into guest '%s'." % vm.name,
+                              logging.info)
+        timeout = float(params.get("login_timeout", 240))
+        session = vm.wait_for_login(timeout=timeout)
+        logging.info("log into guest '%s' successfully." % vm.name)
+
+        error_context.context("Try to ping external host.", logging.info)
+        extra_host_ip = utils_net.get_host_ip_address(params)
+        session.cmd_status_output('ping %s -c 5' % extra_host_ip)
```
Comment (lolyu) [inline qemu/tests/slof_greater_lun_id.py]: Why not check the command result status here?

File: qemu/tests/cfg/slof_greater_lun_id.cfg (hunk 2 above)
Comment (YongxueHong) [reply qemu/tests/cfg/slof_greater_lun_id.cfg]: thanks your comment, have updated.

File: qemu/tests/slof_greater_lun_id.py (hunk 3 above)
Comment (YongxueHong) [reply qemu/tests/slof_greater_lun_id.py]: thanks your comment, have updated.

**Comment by lolyu** [comment]
Thanks all.

**Pull request merged**